src/himrepClassifier/classifier.cpp

```diff
@@ -383,11 +382,10 @@ void Classifier::classify(Bottle *blobs, Bottle &reply)
            y_max+=5;
 
         // crop image
-        ImageOf<PixelRgb> croppedImg;
-        croppedImg.resize(x_max-x_min,y_max-y_min);
-        for(int j=0 ; j<y_max-y_min ; j++)
-            for(int i=0 ; i<x_max-x_min ; i++)
-                croppedImg(i,j) = (*image)(x_min+i,y_min+j);
+        ImageOf<PixelRgb> tmp_img = *image;
+        ::cv::Mat mat_tmp_img = toCvMat(tmp_img);
+        ::cv::Mat mat_croppedImg = mat_tmp_img(cv::Rect(x_min,y_min,x_max-x_min,y_max-y_min)).clone();
+        ImageOf<PixelRgb>  croppedImg = fromCvMat<PixelRgb>(mat_croppedImg);
```

Comment:
I think this is the safest way of doing the cropping, it can be copy/pasted without breaking things.
It can also be reduced to one line if necessary (but it is less readable and you can not reuse the intermediate variables).

Reply:
Awesome 👍